packages/cli/src/test/integration/integration_test.ts

```diff
@@ -503,8 +516,16 @@ suite('polymer shop', function() {
       } else {
         // Cloning and installing takes a minute
         this.timeout(2 * 60 * 1000);
-        const ShopGenerator = createGithubGenerator(
-            {owner: 'Polymer', repo: 'shop', githubToken, branch: '3.0'});
+        const ShopGenerator = createGithubGenerator({
+          owner: 'Polymer',
+          repo: 'shop',
+          githubToken,
+          branch: '3.0',
+          installDependencies: {
+            bower: false,
```

Comment:
do you need the explicit bower:false? why not just npm:true?

Reply:
npm and bower are always run unless you disable them: https://github.com/yeoman/generator/blob/v1.0.1/lib/actions/install.js#L129

Reply:
ah cool